### DISCUSSION TITLE
Error in shipping frontend to elasticsearch

### DISCUSSION THREAD

**Discussion by Bhashitha19** [learndevopsonline/learndevopsonline#1959] [Q&A]
![Screenshot (55)](https://github.com/user-attachments/assets/2b0ece77-ae9e-4512-9e38-cdbbbab19969)

**Comment by chaithanya1812** [comment]
Instance is stoped!!
![image](https://github.com/user-attachments/assets/2d44415e-0ba1-46e5-a3e3-b849f3d3f459)

**Reply by Bhashitha19** [reply]
oh sorry! please check it once again

**Reply by chaithanya1812** [reply]
New public-Ip??

**Reply by Bhashitha19** [reply]
ip: 18.208.152.238

**Reply by chaithanya1812** [reply]
Hi @Bhashitha19 , Issue was ansible needs botocore boto3. I installed it.
![image](https://github.com/user-attachments/assets/07f63d72-321b-4f86-a10c-961e3a9517a5)

## yum install python3.12-pip -y
## pip3.12 install botocore boto3

# after installing main Issue is parameter not found.
![image](https://github.com/user-attachments/assets/86dee124-6afa-404c-8abf-2249e756655e)
